### PR TITLE
Fix docs for running locally

### PR DESCRIPTION
**Pull request by ultraschuppi**
### What does this PR do?

fixes the readme for non-dux-people to run the frontend locally

### Steps for testing:

- burn down your machine, resetup from scratch - or just buy a new one
- checkout repo
- blindly follow the instructions for local development

### Add a GIF:
does not work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +48,7 @@ _Requires node version >= v11.15.0_
 9. Set `NEXT_PUBLIC_MIXPANEL_DEV` to the valid Mixpanel dev environment API key
 10. Set `NEXT_PUBLIC_MIXPANEL_PROD` to the valid Mixpanel prod environment API key
 11. Set `GOERLI_FORK_API_KEY` for the API key
-12. Set `ETHERSCAN_KEY` for ethers provider to use
```
Comment (rafinskipg) [inline README.md:37]: hey @ultraschuppi , i would leave the ETHERSCAN_KEY on the .env variables config section too

**Review by adamgoth — COMMENTED** [review]
Thanks for this @ultraschuppi! Appreciate you bringing attention to some issues non-DUX folks have when trying to run the app. Based on your feedback, I think we can make some additional improvements that may not require a user to sign up for their own etherscan key. The team is going to discuss it and then we'll make the necessary changes.

File: README.md (hunk 1 above)
Comment (ultraschuppi) [reply README.md:37]: good point, shall i just revert the last commit?

**Comment by ultraschuppi** [comment]
> 

that's great news. i am still confident this little PR will have a positive impact - even if it gets superseded by an even better approach in short term future

File: README.md (hunk 1 above)
Comment (adamgoth) [reply README.md:37]: Yes, if you could that'd be great. Then we can merge.

Comment (ultraschuppi) [reply README.md]: done

**Pull request merged**